Soomla/Assets/Plugins/Soomla/Core/Config/android/SoomlaManifestTools.cs

```diff
@@ -21,7 +21,11 @@ public static void GenerateManifest()
             // only copy over a fresh copy of the AndroidManifest if one does not exist
             if (!File.Exists(outputFile))
             {
-                var inputFile = Path.Combine(EditorApplication.applicationContentsPath, "PlaybackEngines/androidplayer/AndroidManifest.xml");
+#if UNITY_5_2
```

Comment:
Maybe it's better to inverse "#if"? Otherwise, we'll need to do an update on the every next Unity3d release. Is it possible?